packages/amplify_flutter/lib/amplify.dart

```diff
@@ -172,8 +170,12 @@ class AmplifyClass extends PlatformInterface {
           underlyingException: e.toString());
     }
     try {
+      // Read the version from local pubspec file.
+      File pubspecFile = new File('pubspec.yaml');
```

Comment:
callout: dk if there is anything special to consider with the path, reading file, gets confusing and not totally aware of gotchas that might exist in flutter env.

Reply:
This is throwing an exception for me in the example projects in amplify_flutter and amplify_auth_cognito, on iOS and Android 🤔 

```
FileSystemException (FileSystemException: Cannot open file, path = 'pubspec.yaml' (OS Error: No such file or directory, errno = 2))
```

Reply:
If this is working for everyone else, I am assuming something is messed up locally for me. @cshfang did this run fine for you?

Reply:
Not related to that exception -

I see the advantage of reading it from pubspec.yaml, and I think that is probably better long term. But if there are any doubts about doing it that way right now, we could just go with the much less sophisticated solution of a find/replace just like all the other version numbers.

Amplify.dart
```dart
final _version = '0.1.4';

class AmplifyClass extends PlatformInterface {
  ...
}
```

bump_version.dart
```dart
await Process.run('/bin/sh', [
  '-c',
  'sed -i "" "s/final _version = \'$oldVersion\'/final _version = \'$newVersion\'/g" packages/amplify_flutter/lib/amplify.dart'
]);
```

Reply:
I was able to run the base example project but did not try any of the others

Reply:
I did this in last commit with the small tweak that I did not add _version as top-level variable just to avoid changing this file here.